oarepo_oidc_einfra/resources.py

```diff
@@ -67,18 +89,94 @@ def upload_dump(self):
                 "message": "Content-Type must be application/json",
             }, 400
 
-        dump_url = current_app.config["EINFRA_DUMP_DATA_URL"]
-        now = datetime.utcnow().strftime("%Y-%m-%d-%H-%M-%S")
-        dump_path = f"{dump_url}/{now}.json"
-
-        location = PyFSFileStorage(dump_path)  # handles both filesystem and s3
-        with location.open(mode="wb") as f:
-            f.write(request.data)
-
+        dump_path = store_dump(request.data)
```

Comment:
What if dump wasn't stored (s3 /network exception / malformed request data)?

Reply:
Could there be any verification of request data integrity & authenticity, before storing?

Reply:
Unfortunately Perun does not send any checksum, co no way to check this apart from parsing - but the json can be large and parsing could take time.
Not checking here would mean that in case of error the data would not be parsed in background task, reported to glitchtip and user can re-trigger sync (or it would happen automatically on next change/next day)